static/app/views/settings/organizationMembers/organizationMembersList.spec.tsx

```diff
@@ -683,7 +623,11 @@ describe('OrganizationMembersList', function () {
         context: TestStubs.routerContext([{organization: org}]),
       });
 
-      expect(screen.getByTestId('invite-banner')).toBeInTheDocument();
+      await act(tick);
```

Comment:
i suspect these act(ticks) are because the useEffect has no dependencies, but it might also be something else. Try removing them after adding the useEffect dependency array

Reply:
it still fails and i'm getting the act() error again